modules/markup/renderer.go

```diff
@@ -163,6 +177,18 @@ type nopCloser struct {
 
 func (nopCloser) Close() error { return nil }
 
+func renderIFrame(ctx *RenderContext, renderer Renderer, input io.Reader, output io.Writer) error {
+	_, err := io.WriteString(output, fmt.Sprintf(`<iframe src="%s%s/%s/render/%s/%s" name="ifd"
```

Comment:
can you add the sandbox attribute (more details here: https://www.w3schools.com/tags/att_iframe_sandbox.asp). The attribute will prevent JS inside iframe from writing to parent window.

Reply:
Done.